tests/restv1/PayPal/CartCest.php

```diff
@@ -77,6 +77,7 @@ public function should_allow_getting_cart_for_post( Restv1Tester $I ) {
 	/**
 	 * It should allow getting cart for post without post ID.
 	 *
+	 * @skip This isn't an endpoint we're using yet.
```

Comment:
Should we leave a @todo to revise this later on?

Reply:
Any skipped tests will be reviewed in the next testing pass, so those don't need todo markers.

Reply:
In this specific test's case, we will be looking at this in the future PayPal work so it can stay in place so we can take advantage of it when it's ready.